src/index.ts

```diff
@@ -82,14 +87,14 @@ export function createFluentVue(options: FluentVueOptions): FluentVue {
           },
         })
 
-        vue2.prototype.$t = function (key: string, value?: Record<string, FluentVariable>) {
+        vue2.prototype[globalT] = function (key: string, value?: Record<string, FluentVariable>) {
           return getContext(rootContext, this).format(key, value)
         }
-        vue2.prototype.$ta = function (key: string, value?: Record<string, FluentVariable>) {
+        vue2.prototype[globalTa] = function (key: string, value?: Record<string, FluentVariable>) {
           return getContext(rootContext, this).formatAttrs(key, value)
         }
 
-        vue2.directive('t', createVue2Directive(rootContext))
+        vue2.directive(rawGlobalT, createVue2Directive(rootContext))
       }
 
       (vue as Vue).component('i18n', component)
```

Comment:
Allow to configure component name too

Reply:
Pushed the ability to change the name of the component.